styles.js

```diff
@@ -9,47 +9,54 @@ export default createGlobalStyle`
 
   body {
     margin: 0;
-    font-family: fantasy;
+    font-family: sans-serif;
+    background-color: #f3eedd;
   }
-  h1 {text-align: center;
+  h1 {
+    text-align: center;
+    color: #506c7f;
     font-size: 0.5rem
-
   }
+  
   h2 {text-align: center;
     font-size: 0.5rem
 
   }
-  ul {
-  list-style: none;
+
+  h3 {
+    color: #506c7f;
   margin: 0;
-  padding: 0;
-}
-li {
-  border-bottom: 1px solid black;
-  margin-bottom: 5px;
-  font-weight: bold;
-  color: #506C7F
-  
-}
+  }
+
+  p {
+  color: #506c7f;
+  margin: 0;
+  }
+
+  ul {
+    padding: 0;
+    margin: 0;
+  }
+
 a {
   
   text-decoration: none;
 
 }
-input {
-  display: block;
-  margin: auto;
+image {
+  max-width: 100%;
+  height: auto;
 }
 
-  .container {
+
+
+
+.buttonlink {
   display: flex;
-  text-align: center;
-  justify-content: space-between;
-  border: 3px solid navy;
-  border-radius:10px;
-  margin: 20px;
+  justify-content: flex-end;
 }
 
+
 .box {
```

Comment:
you should remove all css classes and work with styled components instead for consistency